src/hardware/voodoo.cpp

```diff
@@ -5672,12 +5672,12 @@ static void register_w(uint32_t offset, uint32_t data)
 		case texBaseAddr_1:
 		case texBaseAddr_2:
 		case texBaseAddr_3_8:
-			if (chips & 2)
+			if ((chips & 2) != 0)
```

Comment:
I'm not against these changes, I just had the impression we were okay with implicit true/false checks when using integer expressions in `if` statements. But happy to do these checks explicitly in the future.

Reply:
The voodoo code mixes bools and ints both ways: like storing an int as a bool and vice-versa.  So there were probably some PVS or MSVC warnings about it and I just mass-fixed it all to make it all explicit.

I original started to switch based on one of your original code review comments (where the code was assessing integers in an if/else statement) :-)

If the code is obvious or self-documenting, then I don't really mind either way (and it's really up to the reviewer what they want to allow or not allow).  So.. your call!

Perhaps if this were new code (looking at the code snippet above), we'd be using a bitview.. and have something like:

```c++
   if (chips_register.has_texture) {
       ...
   }
```